mkt/webapps/cron.py

```diff
@@ -124,11 +124,11 @@ def clean_old_signed(seconds=60 * 60):
     for nextroot, dirs, files in walk_storage(root):
         for fn in files:
             full = os.path.join(nextroot, fn)
-            age = now() - storage.modified_time(full)
+            age = now() - private_storage.modified_time(full)
             if age.total_seconds() > seconds:
                 log.debug('Removing signed app: %s, %dsecs old.' % (
                     full, age.total_seconds()))
-                storage.delete(full)
+                private_storage.delete(full)
```

Comment:
For a second I thought it was wrong, but no, these are reviewer signed packages, so carry on :)